app/controllers/sessions_controller.rb

```diff
@@ -14,22 +14,7 @@ def github
     login(role_id: custom_role_or_default(role_id))
   end
 
-  def google
-    return show_login_restriction unless allowed_to_login
-    login(role_id: custom_role_or_default(Role::VIEWER.id))
-  end
-
-  def ldap
-    return show_login_restriction unless allowed_to_login
-    login(role_id: custom_role_or_default(Role::VIEWER.id))
-  end
-
-  def gitlab
-    return show_login_restriction unless allowed_to_login
-    login(role_id: custom_role_or_default(Role::VIEWER.id))
-  end
-
-  def bitbucket
+  def omniauth_callback
```

Comment:
github is 1-off so keeping it separate for now, but eventually it should go through here too